Makefile

```diff
@@ -230,17 +241,22 @@ install-resources:
 	kubectl apply -k deploy/rbac
 	kubectl apply -f deploy/manager/service-account.yaml -n $(KIND_NAMESPACE)
 
+IS_HOSTED ?= false
+E2E_PROCS=20
+
 .PHONY: e2e-test
 e2e-test: e2e-dependencies
-	$(GINKGO) -v --procs=20 $(E2E_TEST_ARGS) test/e2e
+	$(GINKGO) -v --procs=$(E2E_PROCS) $(E2E_TEST_ARGS) test/e2e -- -is_hosted=$(IS_HOSTED)
 
 .PHONY: e2e-test-coverage
 e2e-test-coverage: E2E_TEST_ARGS = --json-report=report_e2e.json --label-filter='!hosted-mode && !running-in-cluster' --output-dir=.
 e2e-test-coverage: e2e-run-instrumented e2e-test e2e-stop-instrumented
 
 .PHONY: e2e-test-hosted-mode-coverage
-e2e-test-hosted-mode-coverage: E2E_TEST_ARGS = --json-report=report_e2e_hosted_mode.json --label-filter="hosted-mode && !running-in-cluster" --output-dir=.
+e2e-test-hosted-mode-coverage: E2E_TEST_ARGS = --json-report=report_e2e_hosted_mode.json --label-filter="hosted-mode || hosted-as-well && !running-in-cluster" --output-dir=.
```

Comment:
Why do we need a second label for hosted tests?

Reply:
> ...Spec selection can be handled in `e2e_suite_test.go`.

Actually, it seems I might be wrong on that. But it seems the goal is to get `case38` to run? So maybe instead of the flag we might be able to just parse the labels in `case38`. Untested, but something like:
```
if Label("hosted-mode").MatchesLabelFilter(GinkgoLabelFilter()) {
```
ref: https://onsi.github.io/ginkgo/#spec-labels

It'd depend on how exactly Ginkgo handles setting up the nodes, though.

Reply:
Operator Policy tests are tested in normal mode as well